### PR TITLE
Update art title and tileset

### PR DESCRIPTION
**Pull request by Mikotey**
# Description

Adding new art and sfx to the project

## Related issue(s)

https://github.com/Saplings-Projects/Birthday-2024/issues/15

## List of changes

A more detailed list of the changes.
- Updated the art for tileset
- Added title logo
- Animated title logo
- Added simple bgm
- Added button sfx

### CODE REVIEW

**Comment by Mikotey** [comment]
Kept the commented out bgm fade-in in case we reconsider it. I think it sounds better with no fade-in.
You can uncomment it and try it. Lemme know which you prefer

**Comment by Casulono** [comment]
> Kept the commented out bgm fade-in in ScreenManager. Just in case we reconsider it. I think it sounds better with no fade-in. You can uncomment it and try it. Let me know which you prefer.

Tried both versions and I don;t really hear much difference

File: Birthday-2024-Project/Scripts/GridLogic.gd [hunk 1]
```diff
@@ -4,7 +4,7 @@ class_name GridLogic
 const MAX_HEIGHT : int = 4 #doubled for negative
 const MAX_WIDTH : int = 5 #doubled for negative
 const TILE_ATLAS_OPEN = Vector2i(1, 0)
-const TILE_ATLAS_OCCUPIED = Vector2i(1, 0)
+const TILE_ATLAS_OCCUPIED = Vector2i(2, 0)
```
Comment (Casulono) [inline Birthday-2024-Project/Scripts/GridLogic.gd:7]: what is the purpose of this change?

**Review by Casulono — APPROVED** [review]

File: Birthday-2024-Project/Scripts/GridLogic.gd (hunk 1 above)
Comment (Mikotey) [reply Birthday-2024-Project/Scripts/GridLogic.gd:7]: It changes which tile is used in the tileset. This changes the "occupied" tiles back to using the 3rd tile

**Comment by Mikotey** [comment]
> > Kept the commented out bgm fade-in in ScreenManager. Just in case we reconsider it. I think it sounds better with no fade-in. You can uncomment it and try it. Let me know which you prefer.
> 
> Tried both versions and I don;t really hear much difference

Oh, forgot to reply to this. In order to properly set the tween it needs to initialize the AudioPlayer to some negative decibel value, like -80 db
